src/Microsoft.NET.Sdk.Razor/build/netstandard2.0/Sdk.Razor.CurrentVersion.targets

```diff
@@ -93,7 +93,7 @@ Copyright (c) .NET Foundation. All rights reserved.
     <RazorGenerateIntermediateOutputPath Condition="'$(RazorGenerateIntermediateOutputPath)'==''">$(IntermediateOutputPath)Razor\</RazorGenerateIntermediateOutputPath>
     
     <!-- File name (without extension) of the assembly produced by Razor -->
-    <RazorTargetName Condition="'$(RazorTargetName)'==''">$(TargetName).PrecompiledViews</RazorTargetName>
+    <RazorTargetName Condition="'$(RazorTargetName)'==''">$(TargetName).Views</RazorTargetName>
```

Comment:
Can we make this `.Razor`? or something like that?

The idea is that the **MVC** package sets it to views.